projects/models.py

```diff
@@ -1,5 +1,5 @@
 from django.db import models
-from django.forms.models import ChoiceField
```

Comment:
The ChoiceField is part of the Form model https://docs.djangoproject.com/en/1.8/ref/forms/fields/ I think if we want to use it we need to create a new class that inherits from model.Form, otherwise the difficulty field is not shown in /admin.